packages/terra-clinical-item-display/package.json

```diff
@@ -28,6 +28,7 @@
     "react-intl": ">=2.8.0, <6.0.0"
   },
   "dependencies": {
+    "@cerner/terra-docs": "^1.9.0",
```

Comment:
Why was this dependency added?

Reply:
It was added to use the `Notice` documentation component here: https://github.com/cerner/terra-clinical/pull/848/files#diff-758c4f2b62d9158096f8b324e0d40867fa7c94b0c2b39d6fa93e50b7ed3060faR2